src/components/Slider/Slider.jsx

```diff
@@ -0,0 +1,242 @@
+    return (
+      <SliderProvider {...providerProps}>
+        <div
+          className={bem("", { disabled, "value-shown": showValue }, className)}
+          data-testid={dataTestId}
+          id={id}
+          ref={mergedRef}
+        >
+          <SliderInfix kind="prefix" />
+          <SliderBase className={consumerBem("base")} />
+          <SliderInfix kind="postfix" />
+        </div>
+      </SliderProvider>
+    );
+  }
+);
+
+Slider.sizes = SIZES_BASIC;
+Slider.colors = COLORS_BASIC;
+
+Slider.propTypes = {
+  // ------ SliderBase props
+  /**
+   * Define a string that labels the current element (Slider)
+   */
+  ariaLabel: PropTypes.string,
+  /**
+   * ElementId of Node that have the text needed for labeling the current element (Slider)
+   */
+  ariaLabeledBy: PropTypes.string,
+  /**
+   * Custom `class name` to be added to the component-root-node
+   */
+  className: PropTypes.string,
+  /**
+   * Custom base of class names for add to component-root and internal-components nodes
+   *  - check Dev Tools for found available Custom/Consumer Classes
+   */
+  classNameBase: PropTypes.string,
+  /**
+   * Color Mode (primary/positive/negative) of the component (Slider)
+   */
+  color: PropTypes.oneOf(Object.values(Slider.colors)),
+  /**
+   * Unique TestId - can be used as Selector for integration tests and other needs (tracking, etc)
+   */
+  "data-testid": PropTypes.string,
+  /**
+   * If set to true, Component (Slider) will be disabled
+   *  - impossible to change value of component (Slider)
+   *  - visual changes (opacity)
+   */
+  disabled: PropTypes.bool,
+  /**
+   * Attribute `id` to be added to the component-root-node
+   */
+  id: PropTypes.string,
+  /**
+   * If true switch slider to RRange mode (two Thumbs)
+   */
+  isRange: PropTypes.bool,
+  /**
+   * Max range value of the component (Slider)
+   */
+  max: PropTypes.number,
+  /**
+   * Min range value of the component (Slider)
+   */
+  min: PropTypes.number,
+  /**
+   * Optional onChange callback (for outer trigger or Controlled mode)
+   * - required in Controlled Mode
+   */
+  onChange: PropTypes.func,
+  /**
+   * Step for Component (Slider) changes
+   */
+  step: PropTypes.number,
+  /**
+   * Always show `value` instead of Tooltip
+   */
+  showValue: PropTypes.bool,
+  /**
+   * Size small/medium/large of the component (Slider)
+   */
+  size: PropTypes.oneOf(Object.values(Slider.sizes)),
+  /**
+   * Current/selected value of the range of the Component (Slider)
+   *   - should be used in Controlled Mode only
+   *   - in isRange mode should be an array of two numbers
+   */
+  value: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.number), PropTypes.number]),
+  /**
+   * Default `value` if value not specified
+   *  - should be used in NON-Controlled mode to set initial Value
+   *  - in isRange mode should be an array of two numbers
+   */
+  valueDefault: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.number), PropTypes.number]),
```

Comment:
let's change prop name to defaultValue for creating a consistent API with other components props

Reply:
Done!